packages/next-app/src/wagmi/generated.ts

```diff
@@ -1,4 +1,4 @@
-// Generated by @wagmi/cli@0.1.11 on 3/7/2023 at 11:03:25 PM
+// Generated by @wagmi/cli@0.1.11 on 3/8/2023 at 11:53:55 PM
```

Comment:
`#comment` 🙄 starting to agree with @mendesfabio that this file probably shouldn't be committed. OR it shouldn't add this line here, AND it definitely should be in the linter/formatters ignore list.